packages/http/src/mocker/__tests__/HttpMocker.spec.ts

```diff
@@ -229,6 +228,100 @@ describe('HttpMocker', () => {
             expect(response.body).toEqual((selectedExample as INodeExample).value);
           });
         });
+
+        describe('and the resource has an example, but the request does not specify any', () => {
+          it('should return the first viable example', () => {
+            const response = httpMocker.mock({
+              input: mockInput,
+              resource: mockResource,
+              config: { mock: { dynamic: false } },
+            });
+
+            expect(response.body).toBeDefined();
+
+            const selectedExample = flatMap(mockResource.responses, res =>
+              flatMap(res.contents, content => content.examples || []),
+            )[0];
+
+            expect(selectedExample).toBeDefined();
+            expect(response.body).toEqual((selectedExample as INodeExample).value);
+          });
+        });
+
+        describe('and the resource has not an example', () => {
+          const resourceOperation: IHttpOperation = {
+            id: 'id',
+            method: 'get',
+            path: '/test',
+            request: {},
+            responses: [
+              {
+                code: '200',
+                headers: [],
+                contents: [
+                  {
+                    mediaType: 'application/json',
+                    schema: {
+                      type: 'object',
+                      properties: {
+                        name: { type: 'string', examples: ['Clark'] },
+                        middlename: { type: 'string', examples: ['J'], default: 'JJ' },
+                        surname: { type: 'string', default: 'Kent' },
+                        age: { type: ['number', 'null'] },
+                        email: { type: 'string' },
+                        deposit: { type: 'number' },
+                        pet: {
+                          type: 'object',
+                          properties: {
+                            name: { type: 'string', examples: ['Clark'] },
+                            middlename: { type: 'string', examples: ['J'], default: 'JJ' },
+                          },
+                        },
+                      },
+                      required: ['name', 'surname', 'age', 'email'],
+                    },
+                    encodings: [],
+                  },
+                ],
+              },
+            ],
+          };
+
+          const response = httpMocker.mock({
+            input: mockInput,
+            resource: resourceOperation,
+            config: { mock: { dynamic: false } },
+          });
+
+          describe('the property has an example key', () => {
+            it('should return the example key', () => expect(response.body).toHaveProperty('name', 'Clark'));
+            describe('the property has an example and a default key', () => {
+              it('should still prefer the example', () => expect(response.body).toHaveProperty('middlename', 'J'));
```

Comment:
specs are about assertions so be assertive! :D 

```suggestion
              it('prefers the example', () => expect(response.body).toHaveProperty('middlename', 'J'));
```